src/forward_sensitivity.jl

```diff
@@ -175,9 +175,11 @@ function ODEForwardSensitivityProblem(f::F,u0,
     mm = f.mass_matrix
   else
     nn = size(f.mass_matrix, 1)
-    mm = similar(f.mass_matrix, 2nn, 2nn)
+    mm = zeros(eltype(f.mass_matrix), (length(p)+1)*nn, (length(p)+1)*nn)
     mm[1:nn, 1:nn] = f.mass_matrix
-    mm[nn+1:2nn, nn+1:2nn] = f.mass_matrix
+    for i = 1:length(p)
```

Comment:
```suggestion
    for i = 1:length(p)+1
```

Reply:
That gives a dimension error.

The way I did it seems to pass tests on rober comparing:

ForwardDiffSensitivity with mass matrix
ForwardSensitivity with mass matrix
ForwardSensitivity with no mass matrix

Reply:
oh I see, you're doing i+1